TombEngine/Objects/Generic/Traps/dart_emitter.cpp

```diff
@@ -19,13 +19,16 @@ namespace TEN::Entities::Traps
 		{
 			DoDamage(LaraItem, 25);
 			DoBloodSplat(item->Pose.Position.x, item->Pose.Position.y, item->Pose.Position.z, (GetRandomControl() & 3) + 4, LaraItem->Pose.Orientation.y, LaraItem->RoomNumber);
+			
+			if (item->TriggerFlags)
 			Lara.PoisonPotency += 1; // Was 160 with the total poison potency later shifted right by 8 when applied to Lara's health. The effect was that each dart contributed a mere fraction to the potency. @Sezz 2022.03.09
```

Comment:
I'd also customize the amount of damage and poisoning that darts can inflict. For example, by default you can set damage to 25 (by the way, you can make it constexpr DEFAULT_DART_DAMAGE), but if OCB is something else, then use this value.

Negative values may mean that darts only decrease specified damage from Lara, while positive values may also add poison to her. For example, dart emitter with OCB 120 will decrease health by 120 and also poison her, while OCB -120 may just decrease her health.

Reply:
That's a good idea. I made negative numbers to poison Lara and positive numbers for non-poison damage